upgrade to Selenium 3.5.3

niQo opened the pull request:
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/docker-selenium/blob/master/CONTRIBUTING.md#contributing-code-to-selenium)

Elufimov commented:
I think it would be better to upgrade to https://selenium-release.storage.googleapis.com/3.5/selenium-server-standalone-3.5.1.jar

niQo commented:
@Elufimov updated to 3.5.1

Elufimov commented:
It was quick! I hope merge will be soon.

Drimix20 commented:
Version of gecko driver for firefox browser is not needed to be updated? Currently is used 0.17.0 version (see ARG GECKODRIVER_VERSION=0.17.0 in NodeFirefox/Dockerfile) but gecko driver has already released v0.18.0.

niQo commented:
@Drimix20 geckodriver v0.18.0 have already been updated, see https://github.com/SeleniumHQ/docker-selenium/commit/3d29e0acd30050feadc4a937814d71b78e0e73ab#diff-79a170fb54c3005e51ce1510022006be

Drimix20 commented:
@niQo Can you look at master branch into NodeFirefox/Dockerfile (not Dockerfile.txt) [link](https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeFirefox/Dockerfile) there is really configured variable
#============
# GeckoDriver
#============
ARG GECKODRIVER_VERSION=0.17.0

niQo commented:
@Drimix20 Dockerfiles are generated from Dockerfile.txt 
see generate.sh

barancev commented:
FYI: version 3.5.3 is available.

ddavison commented:
hey @niQo , thanks for this!

Three favors:

1. Would you squash your commits so i can cherry pick them?
2. Would you rename `argon` to `astatine`?  (we just used argon in 3.5.0)
3. Would you perhaps update your user email and username in your local git configuration?  your commits are showing up as an unknown user.

<img width="353" alt="screen shot 2017-08-30 at 2 59 12 pm" src="https://user-images.githubusercontent.com/2972876/29889904-d54d46c8-8d93-11e7-808c-8761e646c78e.png">

westy92 commented on Hub/Dockerfile:
```diff
@@ -2,7 +2,7 @@
 # NOTE: DO *NOT* EDIT THIS FILE.  IT IS GENERATED.
 # PLEASE UPDATE Dockerfile.txt INSTEAD OF THIS FILE
```
Are you supposed to modify Dockerfile.txt instead?

niQo commented:
@ddavison I've added username/email, changed to astatine and  squashed 3.5.3 in this commit https://github.com/SeleniumHQ/docker-selenium/pull/538/commits/c52d14a7e162424ff566bee97ccfe1fa0b47e995

jsa34 commented:
Any news on update to release 3.5.3?

ddavison commented:
fixed in b955fcc  will close issue when released to docker hub
